src/components/timecard/add-timecard-period/AddTimeCardPeriod.jsx

```diff
@@ -36,9 +33,6 @@ const AddTimeCardPeriod = ({ timecardEmpty }) => {
               </Link>
             </div>
           </div>
-          <p className="govuk-body-m govuk-secondary-text-colour">
```

Comment:
The `govuk-secondary-text-colour` style isn't being used anywhere anymore, do we still want to keep it? Happy to leave it if we think we will use it elsewhere

Reply:
removed from common.scss